sdk/resourcemanager/Azure.ResourceManager.Core/src/ExtensionResourceContainer.cs

```diff
@@ -31,7 +31,7 @@ protected ExtensionResourceContainer(OperationsBase operations)
         /// <param name="operations"> The operations to copy the client options from. </param>
         /// <param name="parentId"> The resource Id of the parent resource. </param>
         protected ExtensionResourceContainer(OperationsBase operations, ResourceIdentifier parentId)
```

Comment:
this comment is true for quite a few cases, can we modify the constructor to take in a ClientContext instead of operations?